ui/package.json

```diff
@@ -41,11 +41,13 @@
     "clsx": "^2.1.0",
     "cmdk": "^1.0.0",
     "lucide-react": "^0.428.0",
+    "next-themes": "^0.3.0",
```

Comment:
Why is this also needed as a top-level component (and not e.g. a dependency of sonner)?